packages/synapse-react-client/src/components/SynapseForm/SynapseFormSubmissionsGrid.tsx

```diff
@@ -472,8 +472,6 @@ export class SynapseFormSubmissionGrid extends React.Component<
                     color="primary"
                     sx={{
                       padding: '0.75rem 1.5rem',
-                      // TODO: remove this line after we have removed bootstrap
-                      '&:hover': { color: 'common.white' }, // otherwise tab-focus.less overrides color
```

Comment:
Consolidate this styling into WideButton -- may be able to remove entirely after changes to DefaultTheme in #292 are merged